transiXNOR/plotting.py

```diff
@@ -261,6 +262,33 @@
 		vds_cur.append(vbg_cur)
 	np.save(model_path+'/current', np.array(vds_cur))
 
+if (QV_CALCULATION):
 
+	e0 = 8.854e-12  # (F/m)
+	er = 25  # for HfO2
+
+	fn_band_diag = './data/phi_' + voltage + '.npy'
+	band_diag = np.load(fn_band_diag)
+	# print(band_diag.shape)
+
+	band_diag = np.reshape(band_diag, (-1, Nx))
+	# print('band diagram after reshape:', band_diag)
+
+	Q_top = []
+	Q_bottom = []
+	for y in enumerate(gridy):
```

Comment:
You should ONLY sum up the charges in the gate region. Check line 77 to line 79 in device.py to see where is the gate region.